package-lock.json

```diff
@@ -6680,6 +6685,7 @@
         "express": "4.17.1",
         "form-urlencoded": "3.0.0",
         "http-errors": "~1.6.3",
+        "jquery": "^3.4.1",
```

Comment:
jquery is still in package-lock.json. Please make sure to update it.

Reply:
Shouldn'tgit  that automatically get removed after an npm install after removing the package?

Reply:
actually the reason it's still in there is that office-addin-sso uses jquery

Reply:
I think you have to use `npm install <package-name>` or `npm uninstall <package-name>` for that to happen. If you manually modify package.json, I don't think it necessarily happens.

Reply:
npm uninstall jquery does nothing to update package-lock.json